From 51957837575210517f5917d1cebcda1fe90f509b Mon Sep 17 00:00:00 2001
From: Tomaz Canabrava <tomaz.canabrava@gmail.com>
Date: Fri, 19 Aug 2016 12:43:24 -0300
Subject: [PATCH] Fix build by undefining the max macro

The max macro was being defined in the c++ namespace,
conflicting with the max C++ function call, breaking the
build.

Signed-off-by: Tomaz Canabrava <tomaz.canabrava@gmail.com>
---
 krfb/rfb.h | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/krfb/rfb.h b/krfb/rfb.h
index fa94eda..1df4836 100644
--- a/krfb/rfb.h
+++ b/krfb/rfb.h
@@ -8,6 +8,10 @@
 
 #include "rfb/rfb.h"
 
+#ifdef max
+#undef max
+#endif
+
 #undef TRUE
 #undef FALSE
 
